### ISSUE TITLE
package dependency on old webcharts = crash

### ISSUE THREAD

**Issue by rbudrevich** [RhoInc/web-codebook#276]
v1.4.0/v1.4.1 explicitly forces a dependency on webcharts 1.7 in package.json, which causes havoc when it gets to rhomap - it crashes in webcharts layout() when you try to expand a variable. Letting it use newer webcharts (in my case 1.10), made it happy.

**Comment by jwildfire** [comment]
Good catch. I'll update package.json and do a quick bug fix release later today. 

cc: @emmorris
